test/ee2_workspaceauth_test.py

```diff
@@ -0,0 +1,115 @@
+import unittest
```

Comment:
Have you ever tried cassettes? You can specify the requests in yaml. I tried to use it at https://github.com/kbase/JobRunner/pull/4/files at some point.

Reply:
Were you able to get it to work? That all looks commented out.

Reply:
Yes, but I just didn't have that much use for it in the job runner and using it got rejected

Reply:
I think in this case, I prefer requests_mock. It's maybe a little more code to set up, but it's pretty easy to parameterize and set up handlers to craft responses based on test input. It's also useful to set up explicit error conditions that might be hard to catch with real server calls (like timeouts, bad or incomplete data, etc.)